Formula/neovim.rb

```diff
@@ -0,0 +1,60 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io/"
+  url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
+  sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
+  head "https://github.com/neovim/neovim.git"
+
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "jemalloc"
+  depends_on "libtermkey"
+  depends_on "libuv"
+  depends_on "libvterm"
+  depends_on "lua@5.1"
+  depends_on "luajit"
+  depends_on "msgpack"
+  depends_on "unibilium"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
```

Comment:
Indeed. So is the logic  that anything post snow leopard will build with python automatically since its shipped with macOS ?

I couldn't see that there was any linkage to python in `brew linkage neovim` but perhaps it doesn't show up there. If that isn't the case there is no option to build `--with-python` for anything post snow leopard which I would assume would be the intention if you wanted to offer python script support =).